postgres/client.sls

```diff
@@ -1,17 +1,18 @@
-{% from "postgres/map.jinja" import postgres with context %}
+{%- from "postgres/map.jinja" import postgres with context -%}
+
+{%- set pkgs = [] %}
+{%- for pkg in (postgres.pkg_client, postgres.pkg_libpq_dev) %}
+  {%- if pkg %}
+    {%- do pkgs.append(pkg) %}
+  {%- endif %}
+{%- endfor -%}
 
-{% if postgres.use_upstream_repo %}
```

Comment:
Why do you remove the conditional include? If I set `postgres.use_upstream_repo = False`, makes no sense to me to force a dependency on a state that sets and manages something I explicitely said don't want to use.